tests/unittest_inference.py

```diff
@@ -901,12 +901,12 @@ class D(C):
         """,
             "module",
         )
-        should_be_C = list(ast[0].infer())
-        should_be_D = list(ast[1].infer())
-        self.assertEqual(1, len(should_be_C))
-        self.assertEqual(1, len(should_be_D))
-        self.assertEqual("module.C", should_be_C[0].qname())
-        self.assertEqual("module.D", should_be_D[0].qname())
+        should_be_c = list(ast[0].infer())
+        should_be_d = list(ast[1].infer())
```

Comment:
Was there really a reason to change `C` and `D` to lowercase?

Reply:
pep8 says variable name should be in snake_case.